satisfactory/chat.py

```diff
@@ -2,79 +2,85 @@
+    async def start_handler(self, update: Update, context: CallbackContext):
         logging.info("/start called for chat %s", update.effective_chat.id)
-        self.send_random_quote(context)
+        await self.send_random_quote(context)
 
     @Summary("satisfactory_bot_quote_handler", "Time spent running quote_handler").time()
-    def quote_handler(self, update: Update, context: CallbackContext):
+    async def quote_handler(self, update: Update, context: CallbackContext):
         logging.info("/quote called for chat %s", update.effective_chat.id)
-        self.send_random_quote(context)
+        await self.send_random_quote(context)
 
     @Summary("satisfactory_bot_check_images_handler", "Time spent running check_images_handler").time()
-    def check_images_handler(self, update: Update, context: CallbackContext):
+    async def check_images_handler(self, update: Update, context: CallbackContext):
         logging.info("/images called for chat %s", update.effective_chat.id)
-        self.check_images(context)
+        await self.check_images(context)
 
     @Summary("satisfactory_bot_check_news_handler", "Time spent running check_news_handler").time()
-    def check_news_handler(self, update: Update, context: CallbackContext):
+    async def check_news_handler(self, update: Update, context: CallbackContext):
         logging.info("/news called for chat %s", update.effective_chat.id)
-        self.check_news(context)
+        await self.check_news(context)
 
     @Summary("satisfactory_bot_check_images_timer", "Time spent running check_images_timer").time()
-    def check_images_timer(self, context: CallbackContext):
-        self.check_images(context)
+    async def check_images_timer(self, context: CallbackContext):
+        await self.check_images(context)
 
     @Summary("satisfactory_bot_check_news_timer", "Time spent running the check_news_timer").time()
-    def check_news_timer(self, context: CallbackContext):
-        self.check_news(context)
+    async def check_news_timer(self, context: CallbackContext):
+        await self.check_news(context)
 
     @Summary("satisfactory_bot_check_videos_timer", "Time spent running the check_videos_timer").time()
-    def check_videos_timer(self, context: CallbackContext):
-        self.check_videos(context)
+    async def check_videos_timer(self, context: CallbackContext):
+        await self.check_videos(context)
 
     @Summary("satisfactory_bot_send_random_quote_timer", "Time spent running the send_random_quote_timer").time()
-    def send_random_quote_timer(self, context: CallbackContext):
-        self.send_random_quote(context)
-        self.updater.job_queue.run_once(self.send_random_quote_timer, random.randrange(60*60*24, 60*60*24*7, 1))
+    async def send_random_quote_timer(self, context: CallbackContext):
+        await self.send_random_quote(context)
+        (self._app.job_queue.run_once(self.send_random_quote_timer,
```

Comment:
Is the outer `(...)` needed for something?

Reply:
hmm, I didn't add that explictly, so maybe formatting, although I don't think it should be necessary :thinking:

Reply:
😄 Yeah, looks a bit weird, but it shouldn't hurt, so 🤷